src/Integrator/Integrator.cpp

```diff
@@ -49,21 +49,21 @@ Integrator::Integrator() : amrex::AmrCore()
         // These are parameters that are specific to
         // the AMR/regridding part of the code.
         IO::ParmParse pp("amr");
-        pp_query("regrid_int", regrid_int);           // Regridding interval in step numbers
-        pp_query("base_regrid_int", base_regrid_int); // Regridding interval based on coarse level only
-        pp_query("plot_int", plot_int);               // Interval (in timesteps) between plotfiles
-        pp_query("plot_dt", plot_dt);                 // Interval (in simulation time) between plotfiles
-        pp_query("plot_file", plot_file);             // Output file
+        pp_query_default("regrid_int", regrid_int, 2);           // Regridding interval in step numbers
+        pp_query_default("base_regrid_int", base_regrid_int, 0); // Regridding interval based on coarse level only
+        pp_query_default("plot_int", plot_int, -1);               // Interval (in timesteps) between plotfiles (OFF)
+        pp_query_default("plot_dt", plot_dt, -1.0);                 // Interval (in simulation time) between plotfiles (OFF)
```

Comment:
@mongole584 Please indicate in the comment that the default negative value will cause the plot interval or plot dt to be ignored.